src/app/(website)/changelog/[slug]/page.tsx

```
@@ -8,13 +8,13 @@ import { Calendar, ChevronLeft, ChevronRight } from "lucide-react"
 import { getChangelogPostBySlug, getChangelogPosts } from "@/lib/changelog"
 import { getMetadata } from "@/lib/get-metadata"
 import { cn } from "@/lib/utils"
+import Breadcrumbs from "@/components/ui/breadcrumbs"
 import { Button } from "@/components/ui/button"
-import { Link } from "@/components/ui/link"
 import Authors from "@/components/pages/changelog/authors"
 import Categories from "@/components/pages/changelog/categories"
-import SocialShare from "@/components/pages/changelog/social-share"
 import Content from "@/components/pages/content"
 import Date from "@/components/pages/date"
```

Biome (2.1.2)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)